projects/plugins/protect/src/js/components/credentials-needed-modal/index.jsx

```diff
@@ -59,7 +59,7 @@ const CredentialsNeededModal = () => {
 
 			<Text mb={ 3 }>
 				{ __(
-					'Once you’ve entered server credentials, Jetpack will be fixing the selected threats.',
+					'Once you’ve entered server credentials, Jetpack Protect will be fixing the three selected threats.',
```

Comment:
```suggestion
					'Once you’ve entered server credentials, Jetpack Protect will be fixing the selected threats.',
```
Minor regression here with the "three" being reintroduced, probably left over from resolving a merge conflict or something like that.